samples/QuickstartWeatherServer/QuickstartWeatherServer.csproj

```diff
@@ -13,7 +13,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\..\src\ModelContextProtocol\ModelContextProtocol.csproj" />
+        <ProjectReference Include="..\..\src\ModelContextProtocol.Hosting\ModelContextProtocol.Hosting.csproj" />
```

Comment:
Instead of "Hosting", should we consider using "Server" to bring this closer to MCP lingo?